webapp/components/test-file-results.js

```diff
@@ -96,7 +96,7 @@ class TestFileResults extends WPTFlags(LoadingState(PathInfo(
   }
 
   static get observers() {
-    return ['loadData(path, testRuns, structuredSearch, onlyShowDifferences)'];
+    return ['loadData(path, testRuns, structuredSearch)'];
```

Comment:
This is a drive-by change. The signatures no longer match (probably changed at some point).